### DISCUSSION TITLE
Could you please tag a new release?

### DISCUSSION THREAD

**Discussion by garak** [reactphp/reactphp#589] [Q&A] [answered]
There were some incompatibilities with PHP 8.4, recently fixed.
Now PHP 8.4 is out, so we need a new release to use this library.

Thanks in advance.

**Comment by clue** [comment, marked as answer]
Anything you're missing in particular?

* https://github.com/reactphp/promise/releases/tag/v3.2.0
* https://github.com/reactphp/promise-timer/releases/tag/v1.11.0
* https://github.com/reactphp/async/releases/tag/v4.3.0
* https://github.com/reactphp/stream/releases/tag/v1.4.0
* https://github.com/reactphp/dns/releases/tag/v1.13.0
* https://github.com/reactphp/socket/releases/tag/v1.16.0
* https://github.com/reactphp/datagram/releases/tag/v1.10.0
* https://github.com/reactphp/http/releases/tag/v1.11.0

This should have you covered already on day one :shipit: 

I hope this helps! :+1: If so, consider supporting this project, for example by [becoming a sponsor](https://github.com/sponsors/reactphp) ❤️
